testkit/src/main/scala/org/bitcoins/testkit/node/fixture/NodeConnectedWithBitcoind.scala

```diff
@@ -15,9 +16,14 @@ case class SpvNodeConnectedWithBitcoind(
     bitcoind: BitcoindRpcClient)
     extends NodeConnectedWithBitcoind
 
-case class SpvNodeConnectedWithBitcoindV21(
```

Comment:
Why delete the `SpvNodeConnectedWithBitcoindV21` ? Just create a new case class `SpvNodeConnectedWithBitcoindV22`